WordPress/WordPressStatsWidgets/LockScreenWidgets/Configs/LockScreenAllTimePostsBestViewsStatWidgetConfig.swift

```diff
@@ -0,0 +1,56 @@
+import WidgetKit
+
+@available(iOS 16.0, *)
+struct LockScreenAllTimePostsBestViewsStatWidgetConfig: LockScreenStatsWidgetConfig {
+    typealias WidgetData = HomeWidgetAllTimeData
+    typealias ViewProvider = LockScreenMultiStatWidgetViewProvider
+
+    var supportFamilies: [WidgetFamily] {
+        guard AppConfiguration.isJetpack, FeatureFlag.lockScreenWidget.enabled else {
+            return []
+        }
+        return [.accessoryRectangular]
+    }
+
+    var displayName: String {
+        LocalizableStrings.allTimeWidgetTitle
+    }
+
+    var description: String {
+        LocalizableStrings.allTimePreviewDescription
```

Comment:
nit: As the user scrolls through the widget previews before adding one, I think the titles and descriptions could be more descriptive. For this widget, instead of "All Time" as a title it could be `All Time Posts & Views` and the description could also mention that it's specifically for Posts & Views, otherwise the descriptions don't seem to give enough information to understand what's being shown (as they're the same for all the All Time widgets). 

The iOS Weather app, for example, has different titles and descriptions for each widget.

Reply:
I think that's a good point. We want to be as clear as possible. I'll add additional descriptions.

Reply:
Thanks! The new titles look good!